backend/api/game_test.go

```diff
@@ -10,14 +10,13 @@ import (
 	"time"
 
 	"github.com/jak103/usu-gdsf/auth"
-	"github.com/jak103/usu-gdsf/db"
 	"github.com/jak103/usu-gdsf/models"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	_db, _ = db.NewDatabaseFromEnv()
+	// _db, _ = db.NewDatabaseFromEnv()
```

Comment:
This is needed to add games to the db. You should run the tests to check that your changes are working.

Reply:
Sorry Kris, i just saw it. No, this is already declared in the package in another file so there is same variable in a package in multiple file which turned out is not acceptable in go !!

I made it in last minute just before submission so i didn't removed code in rush !!
I will write more test code today and on new pull request i will remove that rather than putting on comment.

Thank you for review, i appreciate it.